core/templates/dev/head/editor/RouterServices.js

```diff
@@ -52,8 +52,24 @@ oppia.factory('routerService', [
 
       $rootScope.$broadcast('externalSave');
 
-      if (newPath === '/preview') {
+      if (newPath.indexOf('/preview/') !== -1) {
```

Comment:
Can this be "=== 0"? Seems stronger, since it requires the string to start with '/preview/'.